docs/poetry.lock

```diff
@@ -1,18 +1,26 @@
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
I doubt it passes the CI, automated dep fixes don't work but I wanted to have a proof :)